Scribbie-Notes/notes-app
# Fix  loading indicator on login and signup button and improve user feedback

### What does this PR do 
   This PR addresses issue #31 by fixing the loading indicator on login and signup button.

### What Changed
- Added a loading spinner to the login and signup buttons.
- Updated the button text to indicate the logging in or signing up process.

### How to Test
1. Click on the login/signup button with valid credentials.
2. Observe the loading spinner appears while processing.
3. Ensure that the button text changes accordingly.

### Related Issues
Fixes #123
